### ISSUE TITLE
Android Build Failed - Could not resolve all artifacts for configuration ':react-native-community_slider:classpath'. 

### ISSUE THREAD

**Issue by Md-Mudassir47** [callstack/react-native-slider#415]
It was running fine till yesterday, I had not updated the package & once it started to throw this issue I then upgraded to the latest version but still the same issue

### Version Details
```
"@react-native-community/slider": "^4.2.0",
"react-native-slider": "^0.11.0",
"react-native": "^0.67.4",
```

### Current Version Of Slider
`"@react-native-community/slider": "^4.3.0",`

```
* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all dependencies for configuration ':app:debugCompileClasspath'.
   > A problem occurred configuring project ':react-native-community_slider'.
      > Could not resolve all artifacts for configuration ':react-native-community_slider:classpath'.
         > Could not find com.facebook.react:react-native-gradle-plugin:.
           Required by:
               project :react-native-community_slider
```

**Comment by Mateoooo** [comment]
I have the same problem. 
`"react-native": "^0.67.2",
`

**Comment by valery-lavrik** [comment]
I have the same problem.
`"react-native": "0.66.4",`

**Comment by BartoszKlonowski** [comment]
@Md-Mudassir47 Are you able to share all the details about:
* react-native version you are using
* Slider version you were using before this error appeared
* Slider version you are using now

This is a duplicate of already reported issue (all have the same root cause): #414 #416,
I'm aware of this problem and I'm actively working on this.

**Comment by Md-Mudassir47** [comment]
> @Md-Mudassir47 Are you able to share all the details about:
> 
> * react-native version you are using
> * Slider version you were using before this error appeared
> * Slider version you are using now
> 
> This is a duplicate of already reported issue (all have the same root cause): #414 #416, I'm aware of this problem and I'm actively working on this.

I've updated it

**Comment by Md-Mudassir47** [comment]
@BartoszKlonowski I managed to fix it by 
Removing `@react-native-community/slider": "^4.2.0`
Replace `import Slider from '@react-native-community/slider';` -> `import Slider from 'react-native-slider';`

**Comment by BartoszKlonowski** [comment]
> @BartoszKlonowski I managed to fix it by Removing `@react-native-community/slider": "^4.2.0` Replace `import Slider from '@react-native-community/slider';` -> `import Slider from 'react-native-slider';`
> 
> Works on both iOS & Android, Android Build is successful 🎉

This is not a fix, this is just using another package, but I'm glad you have your app unblocked 👍

**Comment by luizgallas** [comment]
I am facing the same issue, any solution?

**Comment by minhas-weoveri** [comment]
I am facing the same issue.

**Comment by dipankurXT** [comment]
Same issue here

**Comment by MrQureshi** [comment]
"^4.3.0" this version is unstable for now downgrade version with "4.2.0" and remove caret symbol 
 
 then run command  `npx jetify`

**problem solved**

**Comment by one-god** [comment]
I am facing the same issue.

**Comment by valery-lavrik** [comment]
Yes, just fix the slider connection in the **package.json**

`"@react-native-community/slider": "4.2.4",`

A little higher has already been answered

**Comment by BartoszKlonowski** [comment]
The fix is now released in v4.3.1

**Comment by NguyenPhuongNam99** [comment]
please install version 4.2.4 . It is fix me!

**Comment by AndrewLutsenko** [comment]
![image](https://user-images.githubusercontent.com/129854280/232572335-cbdcd4aa-8d5b-4e90-bd73-e233569adfae.png)
Still not work

react-native v0.70.6
"@react-native-community/slider": "4.3.1",

**Comment by ansarikhurshid786** [comment]
4.3.1 works for react native 0.64.4
